### PR TITLE
New Crowdin translations to review and merge (#5618)

### PR DESCRIPTION
**Pull request by teolemon**
- This is the regular PR that I merged into the wrong branch (crowdin-trigger) 
- I manually reviewed it.
- It's safe to merge.

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]

**Comment by teolemon** [comment]
Wide character in print at /home/runner/perl5/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   '<p>معرفة المزيد عن<site_name> الرسوم البيانية:<a href="/graphs-in-3-clicks">الرسوم البيانية من خلال 3 نقرات</a>(مدونة).</p>'
#     doesn't match '(?^:<<site_name>>)'

#   Failed test 'common: 'text_direction' in 'tr': must be 'ltr' or 'rtl''
#   at t/i18n.t line 55.
Wide character in print at /home/runner/perl5/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   'soldan sağa'
#     doesn't match '(?^:^(ltr|rtl)$)'

**Comment by teolemon** [comment]
Test Summary Report
-------------------
t/ingredients_parsing.t        (Wstat: 256 Tests: 200 Failed: 1)
  Failed test:  69
  Non-zero exit status: 1
Files=38, Tests=3366, 122 wallclock secs ( 0.46 usr  0.10 sys + 179.16 cusr 19.43 csys = 199.15 CPU)
Result: FAIL
Error: Process completed with exit code 1.

**Comment by teolemon** [comment]
@stephanegigandet this time, no idea why it's failing

File: po/common/nb.po [hunk 1]
```diff
@@ -2426,7 +2426,7 @@ msgstr "Søk:"
 
 msgctxt "traces"
 msgid "Traces"
-msgstr "Spor"
+msgstr "Kan inneholde"
```
Comment (teolemon) [inline po/common/nb.po:2429]:
```suggestion
msgstr "Spor"
```

**Pull request merged**